addons/account/models/account_move.py

```diff
@@ -2373,6 +2373,7 @@ class AccountMoveLine(models.Model):
     _name = "account.move.line"
     _description = "Journal Item"
     _order = "date desc, move_name desc, id"
+    _inherit = ['mail.thread']
```

Comment:
Not so fan of this :p 
From what I see, we only need it to be able to call _message_track, right? Why not just making an api.model version of   _message_track in mail to call here, and simply call it from the regular _message_track? (this would thus be the only line of the function)